packages/headless/src/app/engine-configuration.ts

```diff
@@ -150,6 +150,12 @@ export interface AnalyticsConfiguration {
    * Specifies the URL of the current page or component.
    */
   documentLocation?: string;
+  /**
+   * TBD
+   * @internal
+   */
+  trackingId?: string;
```

Comment:
We can start optional, but we'll eventually want `trackingId` to be required. Atm one can pass any string. In Q4, Data Platform will make it possible to create `trackingId` values via the AdminUI, at which point only values registered with the Admin will be valid,

Reply:
Yeah, for now, no need to over fret about this IMO.
Maybe we should coerce backend wise the trackingID tho